pom.xml

```diff
@@ -157,6 +157,19 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Kan maven-source-plugin nu weg uit central-sonatype-release? Lijkt er dubbel in te staan?